ui/src/timeMachine/components/view_options/LineOptions.tsx

```diff
@@ -58,6 +59,7 @@ interface OwnProps {
   geom?: XYGeom
   colors: Color[]
   shadeBelow?: boolean
+  hoverDimension?: 'auto' | 'x' | 'y' | 'xy'
```

Comment:
hoverDimension is an optional argument, but I don't think the dropdown will have anything selected when it is not provided. I think this component should handle setting hoverDimension to 'auto' if none have been provided.